### PR TITLE
Added French Translation

### PR DESCRIPTION
**Pull request by RedBusters**
Not much to say, i just added french translation.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+msgstr "Ouvrir le panneau de configuration"
+
+#: data/ui/ProfileRow.ui:16
+msgid "Remove this profile from the device"
+msgstr "Enlever ce profile du périphérique"
+
+#. Translators: this is used to indicate the active resolution.
+#: data/ui/ResolutionRow.ui:37
+msgid "Active"
+msgstr "Actif"
```
Comment (bentiss) [inline po/fr.po]: If this is about the résolution: `Active`

**Comment by RedBusters** [comment]
Thanks for the review bentiss, i modified the file according to what you said ; )

**Comment by bentiss** [comment]
Sorry for taking so long to come back to you. I still see a few changes not merged, was that intentional?
If you want I can squash your patch together and add one on top of it with m last nitpicks.

**Comment by whot** [comment]
@bentiss, @RedBusters - I'd like to get a new piper release out now, can we get this in within the next few days? My french is ... insufficient for this :)

**Comment by bentiss** [comment]
I took the liberty to rebase and add the last few fixes to your branch.
And I am merging it right now.

**Pull request merged**